### PR TITLE
Change erp domain from erp.ernet.in to erp.ac.in

### PR DESCRIPTION
**Pull request by Ayushk4**

### CODE REVIEW

File: syllabus_scrapper.py [hunk 1]
```diff
@@ -114,4 +115,4 @@ def get_syllabus(subject_code,subject_name):
 
     print(sys.argv)
     if(len(sys.argv)>1):
-        print(get_syllabus(sys.argv[1]))
+        print(get_syllabus(sys.argv[1]))
```
Comment (athityakumar) [inline syllabus_scrapper.py]: Leave a blank line at the EOF, as it was before

**Comment by thealphadollar** [comment]
@Ayushk4 Please implement the requested changes.

**Comment by Ayushk4** [comment]
Ping @thealphadollar

**Review by thealphadollar — APPROVED** [review]
LGTM!

**Pull request merged**